hnn_core/tests/test_viz.py

```diff
@@ -47,9 +55,21 @@ def test_network_visualization():
     with pytest.raises(TypeError, match='src_gid must be an instance of'):
         plot_cell_connectivity(net, conn_idx, src_gid='blah')
 
-    with pytest.raises(ValueError, match='src_gid not in the'):
+    with pytest.raises(ValueError, match='src_gid not a valid cell ID'):
         plot_cell_connectivity(net, conn_idx, src_gid=-1)
 
+    # smoke test interactive clicking
+    del net.connectivity[-1]
+    conn_idx = 15
+    net.add_connection(net.gid_ranges['L2_pyramidal'][::2],
+                       'L5_basket', 'soma',
+                       'ampa', 0.00025, 1.0, lamtha=3.0,
+                       probability=0.8)
+    fig = plot_cell_connectivity(net, conn_idx)
+    ax_src = fig.axes[0]
+    pos = net.pos_dict['L2_pyramidal'][2]
+    _fake_click(fig, ax_src, [pos[0], pos[1]])
+
```

Comment:
Can you add a test to make sure the figure is actually updated? A really simple one would be to grab the title with `fig.texts` and assert that it is different after running `_fake_click`.

Reply:
you're making me work harder for my merges ;-) It's good! Take a look at the last commit